diagnostic.md

```diff
@@ -94,4 +94,6 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+`teamMember['hobby']` === 'exercise'
+`teamMember['name']` === 'Jason Weeks'
```

Comment:
The value of teamMember['name'] is an object, { given: 'Jason', surname: 'Weeks' }.